flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/regression/MultipleLinearRegression.scala

```diff
@@ -107,6 +107,11 @@ class MultipleLinearRegression extends Predictor[MultipleLinearRegression] {
     this
   }
 
+  def setOptimizationMethod(optimizationMethod: String): this.type = {
```

Comment:
Furthermore, if you expose the optimization method, then you should also expose the decay parameter. But to be honest I'm not so sure whether this is the right way to do, because you would have to add to every algorithm every new parameter of the underlying solver. I guess it would be better to expose a parameter where you can set a non-default solver which allows you to set these things directly. What do you think?